pkgs/development/tools/ocaml/js_of_ocaml/3.0.nix

```diff
@@ -11,5 +11,7 @@ stdenv.mkDerivation rec {
 
 	propagatedBuildInputs = [ js_of_ocaml-compiler uchar ];
 
+  patches = [ ./jbuild.diff ];
```

Comment:
This is usually done with `patchShebangs` in the `postPatch` phase.

Reply:
Thank you. That works an is indeed much simpler:
https://github.com/NixOS/nixpkgs/compare/master...ibor:master

Do you need a new pull request?

Reply:
This pull request is fine (although its individual commits could be squashed together for the better), but I don't have the permission to merge yet. (I hope to get it soon.)